yarn.lock

```diff
@@ -2867,6 +2867,11 @@
     unique-filename "^1.1.1"
     which "^1.3.1"
 
+"@grafana/data@6.6.2":
+  version "6.6.2"
+  resolved "https://registry.yarnpkg.com/@grafana/data/-/data-6.6.2.tgz#34695504cd86bc3b07e4dddcf35ebb1b84158199"
+  integrity sha512-561FlVoE8iFaMB4mo6FBCQc/+J/tv77vzjFWcHE2HB1JwDF15IxbABcDzZWTfPy2ypAi7TR3Kc8Lvss5v0l9Qg==
+
```

Comment:
I don't think you want any changes to yarn.lock in this PR

Reply:
nooooo! You are absolutely right.